tsconfig.app.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@vue/tsconfig/tsconfig.web.json",
-  "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
+  "include": ["src/**/*.ts", "env.d.ts", "src/**/*", "src/**/*.vue"],
```

Comment:
Hmm, I think maybe that's not necessary. Was it not working without it? According to documentation typescript files should be included since you specified "src/**/*"

https://www.typescriptlang.org/tsconfig#include
![image](https://user-images.githubusercontent.com/108802861/185335589-82cb4fb0-75f0-4b8b-b0de-bb438e0d34cc.png)

Reply:
It was added when I had a typescript related issue (I don't even remember which one) that was latter solved in different manner - I will clean this up.